### PR TITLE
List Lobstex (LOB)

### PR DESCRIPTION
**Pull request by lobstex**
Author: lobstex@gmail.com
Official project URL: https://lobstex.com/lobstex-1
Official block explorer URL: http://explorer.lobstex.com/

### CODE REVIEW

**Comment by therealredvoid** [comment]
Any response on this?

File: src/main/java/bisq/asset/coins/LobstexCoin.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.params.MainNetParams;
+import org.bitcoinj.params.RegTestParams;
+import org.bitcoinj.params.TestNet3Params;
+
+public class LobstexCoin extends Coin {
+    public LobstexCoin() {
+        super("Lobstex", "LOBS", new DefaultAddressValidator());
```
Comment (blabno) [inline src/main/java/bisq/asset/coins/LobstexCoin.java]: Do not use deprecated DefaultAddressValidator

**Comment by lobstex** [comment]
I've changed the source to reflect @blabno 's change requests.

**Comment by cbeams** [comment]
See my additional fixes in 66002e0.

**Pull request merged**